docs/code_of_conduct.md

```diff
@@ -0,0 +1,14 @@
+# The Systemd Community Conduct Guidelines
+
+This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to systemd. It applies to all “collaborative spaces”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
```

Comment:
According to https://lists.freedesktop.org/mailman/listinfo/systemd-devel, the use of the mailing list is subject to https://www.freedesktop.org/wiki/CodeOfConduct/. Which code of conduct takes precedence there?

Reply:
Good point. Fd.o must have added this fairly recently. Maybe we could ask change to the following:

  Use of all freedesktop.org lists is subject to our Code of Conduct, unless stated otherwise in the list description.

and then add "Use of this list is covered by the [systemd code of conduct](link)." .